client/reader/update-notice/style.scss

```diff
@@ -2,8 +2,8 @@
 // Slides in when there are new posts available
 .reader-update-notice {
 	position: fixed;
-	top: ( 47px + 8px );
 	right: 16px;
+	top: calc(var(--masterbar-height) + 16px);
```

Comment:
I assume the 16px here and 24px below in the media query are the frame border widths at these breakpoints? A good future improvement may be to make variables for those.

Reply:
Sounds good, I'll check if we have a variable, before merging it.

Reply:
No variable: https://github.com/Automattic/wp-calypso/blob/ef0d9368de40386ed467d1a5b05be54869d36cf8/client/sites-dashboard-v2/dotcom-style.scss#L248